src/views/Operations/Firmware/FirmwareAlertServerPower.vue

```diff
@@ -30,22 +30,31 @@
   </b-row>
 </template>
 
-<script>
-import Alert from '@/components/Global/Alert';
+<script setup>
+import { computed } from 'vue';
+import Alert from '@/components/Global/Alert.vue';
+import { ControlStore } from '@/store';
 
-export default {
-  components: { Alert },
-  props: {
-    isServerOff: {
-      required: true,
-      type: Boolean,
-      default: true,
-    },
-  },
-  computed: {
-    isOperationInProgress() {
-      return this.$store.getters['controls/isOperationInProgress'];
-    },
+const controlStore = ControlStore();
+
+defineProps({
+  isServerOff: {
```

Comment:
Looks like the warning message is not being displayed when the system is Powered On